c/validator_utils.h

```diff
@@ -876,9 +876,9 @@ int _load_verification_context(
 
   /* load rollup config cell */
   mol_seg_t rollup_config_hash_seg;
-  if (0 == rollup_version) {
+  if (1 == rollup_version) {
     rollup_config_hash_seg =
-        MolReader_GlobalStateV0_get_rollup_config_hash(&global_state_seg);
+        MolReader_GlobalStateV1_get_rollup_config_hash(&global_state_seg);
   } else {
     rollup_config_hash_seg =
         MolReader_GlobalState_get_rollup_config_hash(&global_state_seg);
```

Comment:
We should read value with `MolReader_GlobalStateV0`, since it is the current version, the `GlobalState` without versioning should be marked as obsoleted.